src/components/book/singleBook.css

```diff
@@ -0,0 +1,95 @@
+.single-book-container {
+  margin: 2.313rem 6.25rem 0;
+  padding: 2rem 9.188rem 1.625rem 1.688rem;
+  border-radius: 4px;
+  background: #fff;
+  border: solid 1px #fff;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  gap: 5px;
+}
+
+.single-book-category,
+.text h2,
+.single-book-author {
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+}
+
+.single-book-category {
+  margin: 0 2.438rem 0 0;
+  opacity: 0.5;
+  font-size: 0.875rem;
+  letter-spacing: normal;
+  color: #000;
+}
+
+.text h2 {
+  margin: 0.188rem 20.188rem 0 0;
+  font-size: 1.375rem;
+  letter-spacing: -0.2px;
+}
+
+.single-book-author {
+  margin: 0.25rem 0 0;
+  font-size: 0.875rem;
+  letter-spacing: normal;
+  color: #4386bf;
+}
+
+.single-book-options ul {
+  display: flex;
+}
+
+.single-book-options ul li {
+  list-style-type: none;
+}
+
+.comments,
+.remove,
+.edit {
+  font-size: 0.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #4386bf;
+}
+
+.comments {
+  margin: 0.188rem 0.938rem 0.125rem 0;
+}
+
+.remove {
+  margin: 0.188rem 0.938rem 0.125rem 1.063rem;
+  cursor: pointer;
+}
+
+.edit {
+  margin: 0.188rem 18.625rem 0.125rem 1.063rem;
+}
+
+.percent {
+  /* display: flex;
+  flex-direction: column; */
```

Comment:
- [ ] Please it will be nice to remove commented code from your code base, as it makes your code base dirty and less readable.

Reply:
Thank you @Mihndim2020. Fixed